src/components/ConnectButton.tsx

```diff
@@ -68,6 +68,10 @@ export const ConnectButton = () => {
                 return <Chip onClick={openChainModal}>Wrong network</Chip>;
               }
 
+              if (!isAllowedToVote) {
+                return <Chip>You are not allowed to vote</Chip>;
```

Comment:
Not sure the best place to do this (prob not a button component) but I think it'd be nice to add some more context, e.g.

"You are not allowed to vote. Only [badgeholders](https://community.optimism.io/docs/governance/citizenship/) are eligible to vote."

Reply:
It's implemented the same way as for wrong network and I don't think we need to give here more context because we don't have enough space here.